var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -36,6 +40,37 @@ class Ncurses(AutotoolsPackage, GNUMirrorPackage):
     patch('patch_gcc_5.txt', when='@6.0%gcc@5.0:')
     patch('sed_pgi.patch',   when='@:6.0')
 
+    @classmethod
+    def determine_version(cls, exe):
+        return Executable(exe)('--version', output=str, error=str).rstrip()
+
+    @classmethod
+    def determine_variants(cls, exes, version):
+        results = []
+        for exe in exes:
+            variants = ''
+            output = Executable(exe)('--libs', output=str, error=str)
+
+            if re.search(r'-ltinfo', output):
```

Comment:
```suggestion
            if '-ltinfo' in output:
```
This removes the `re` dependency